src/foam/lang/Enum.js

```diff
@@ -442,7 +442,11 @@ foam.CLASS({
     {
       name: 'toJSON',
       value: function(value) { return value.ordinal; }
-    }
+    },
```

Comment:
What is the issue? This will cause enums to always be output.

Reply:
The property is enum. It has a value by default and it is not a placeholder. Validation is still showing "required" due to the check on line 196 Validation.js
```javascript
return axiom.isDefaultValue(this[name]) && self.REQUIRED;
```

<img width="151" alt="Screenshot 2025-07-03 at 3 43 23 PM" src="https://github.com/user-attachments/assets/d609faa3-4c9e-4815-9897-ba4ca283a615" />